how-to/create-addon.rst

```diff
@@ -74,7 +74,7 @@ In this case, you will need to add the package to be installed to the
 If your addon *contains* an application
 .......................................
 
-If on the other hand, for example if the application is not available on PyPI,
+On the other hand, if for example, the application is not available on PyPI,
```

Comment:
`If on the other hand the application...`

Reply:
Done